### DISCUSSION TITLE
How to use openpose in Fooocus

### DISCUSSION THREAD

**Discussion by diegotrazzi** [lllyasviel/Fooocus#1214] [Q&A]
Is there a way to use openpose in Fooocus. 
I am trying to build a library of yoga poses and seems extremely challenging with the current either one of the two base models I am using. 
Any ideas, or suggestions? Thanks.

**Comment by aryamanmajumdar** [comment]
I second this. The lack of openpose and reActor are the only reasons I still use A1111. 

The solution right now is to either use pyracanny in foocus. Or use openpose in another ui like A1111 and then do the easier stuff in fooocus.

**Comment by PowZone** [comment]
UP!

**Comment by Rogala** [comment]
[Poses For Free And Easy Editing](https://www.youtube.com/watch?v=8USLRg4ZCgE&t=7s)
